src/nvim/api/vim.c

```diff
@@ -1542,9 +1542,11 @@ Object nvim_load_context(Dictionary dict)
 }
 
 /// Gets the current mode. |mode()|
-/// "blocking" is true if Nvim is waiting for input.
 ///
-/// @returns Dictionary { "mode": String, "blocking": Boolean }
+/// @return Dictionary with the following keys:
+///       - mode: (string) Current mode
+///       - blocking: (boolean) true if Neovim is waiting for input
+///       - cmdpreview: (boolean) true if Neovim is previewing a command
```

Comment:
A bit pedantic, but in the docs when referring to the program we usually call it `Nvim` rather than `Neovim`.

```suggestion
///       - blocking: (boolean) true if Nvim is waiting for input
///       - cmdpreview: (boolean) true if Nvim is previewing a command
```

Reply:
Will keep that in mind, fixed